tests/unit/test_cli.py

```diff
@@ -228,13 +230,54 @@ def __init__(self, outputFormat='text'):
             self.baseUrl = 'baseUrl'
             self.verbose = 'verbose'
 
-    class FakeObject(protocol.ProtocolElement):
-
-        __slots__ = ['id', 'name']
-
-        def __init__(self):
-            self.id = 'id'
-            self.name = 'name'
+    class FakeObject(protocol.message.Message):
```

Comment:
It's a little brittle, but only really dependant on changes to the Protocol Buffers library itself, not ga4gh formats, so I think it's acceptable